deploy/FlapperInit.sol

```diff
@@ -112,4 +133,41 @@ library FlapperInit {
         PipLike(OracleWrapperLike(wrapper).pip()).kiss(wrapper);
         dss.chainlog.setAddress(clKey, wrapper);
     }
+
+    function initSplitter(        
+        DssInstance      memory dss,
+        SplitterInstance memory splitterInstance,
+        SplitterConfig   memory cfg
+    ) internal {
+        SplitterLike    splitter = SplitterLike(splitterInstance.splitter);
+        SplitterMomLike mom      = SplitterMomLike(splitterInstance.mom);
+
+        // Sanity checks
+        require(splitter.vat()     == address(dss.vat), "Splitter vat mismatch");
+        require(splitter.daiJoin() == cfg.daiJoin,      "Splitter daiJoin mismatch");
+        require(splitter.farm()    == cfg.farm,         "Splitter farm mismatch");
```

Comment:
We could also check `mom.splitter() == splitter`.

Reply:
Yes, good point, added.